README.md

```diff
@@ -1 +1,97 @@
 # testcafe-angular-selectors
+
+This plugin provides selector extensions that make it easier to test Angular applications with [TestCafe](https://github.com/DevExpress/testcafe/). These extensions allow you to create a Selector to find elements on the page by **expressions** that are native to Angular applications (like `binding`, `model` etc.. ).
```

Comment:
Now, the word `expression` was renamed to the `descriptor`.
It was done on site (http://www.protractortest.org/#/api?view=ProtractorBy.prototype.binding).
and in the code https://github.com/angular/protractor/blob/master/lib/locators.ts#L132

I will do same.